roles/addons/Lmod/vars/RedHat.yml

```diff
@@ -0,0 +1,3 @@
+---
+Lmod_packages_to_install:
```

Comment:
why a capital lettre for a variable?

Reply:
Hello !
Well, because the tool itself is everywhere using capital L (doc, spec, code, etc). So I tried to comply with that 🙂